content/millennium/dstu2/entities/person.md

```diff
@@ -38,14 +44,14 @@ Search for Persons that meet supplied query parameters:
 
 ### Parameters
 
- Name         | Required?                                         | Type       | Description
---------------|---------------------------------------------------|------------|------------------------------------------------------------------------------------
- `_id`        | No, if populated all other parameters are ignored | [`token`]  | The logical resource id associated with the resource.
- `identifier` | Yes, or `_id`                                     | [`token`]  | The person identifier.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
+ Name         | Required      | Type       | Description
+--------------|---------------|------------|------------------------------------------------------------------------------------
+ `_id`        | Conditionally | [`token`]  | The logical resource ID associated with the resource. This parameter is required if the `identifier` parameter is not used.
+ `identifier` | Conditionally | [`token`]  | The person identifier. This parameter is required if the `_id` parameter is not used.
```

Comment:
Did we want to have examples listed up in the table?  Most of our resources do.

Reply:
```suggestion
 `_id`        | Conditionally | [`token`]  | The logical resource ID associated with the resource. This parameter is required if the `identifier` parameter is not used. Example: `_id=12345`
 `identifier` | Conditionally | [`token`]  | The person identifier. This parameter is required if the `_id` parameter is not used.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
```